File: TransCoder_tokenized_test_set_functions/python/PRINT_STRING_IGNORING_ALTERNATE_OCCURRENCES_CHARACTER.py
def printStringAlternate ( string ) :
    occ = {
    }
    
    for i in range ( 0, len ( string ) ) :
        temp = string [ i ] . lower ( )
        occ [ temp ] = occ . get ( temp, 0 ) + 1
        if occ [ temp ] & 1 :
            print ( string [ i ], end = "" )
    print ( )

File: TransCoder_tokenized_test_set_functions/java/PRINT_STRING_IGNORING_ALTERNATE_OCCURRENCES_CHARACTER.java
static void printStringAlternate ( String str ) {
  int [ ] occ = new int [ 122 ];
  String s = str . toLowerCase ( );
  for ( int i = 0;
  i < str . length ( );
  i ++ ) {
    char temp = s . charAt ( i );
    occ [ temp ] ++;
    if ( occ [ temp ] % 2 != 0 ) System . out . print ( str . charAt ( i ) );
  }
  System . out . println ( );
}